vit-servicing-station-tests/src/common/startup/mod.rs

```diff
@@ -54,19 +55,21 @@ pub fn get_available_port() -> u32 {
     NEXT_AVAILABLE_PORT_NUMBER.fetch_add(1, Ordering::SeqCst)
 }
 
-pub fn quick_start(temp_dir: &TempDir) -> (Server, String) {
+pub fn quick_start(temp_dir: &TempDir) -> Result<(Server, String), ServerBootstrapperError> {
     let (token, hash) = data::token();
 
     let db_path = DbBuilder::new()
         .with_token(token)
         .with_proposals(data::proposals())
-        .build(&temp_dir)
-        .unwrap();
+        .build(&temp_dir)?;
 
     let server = ServerBootstrapper::new()
         .with_db_path(db_path.to_str().unwrap())
-        .start()
-        .unwrap();
+        .start()?;
 
-    (server, hash)
+    if !server.is_up(&hash) {
+        panic!("server is not up");
```

Comment:
Can we have an error for this?

Reply:
i've just converted it to error. we will see in next prs if check_running() would be useful. If yes i will added it